silence/silence.go

```diff
@@ -662,7 +703,7 @@ func (s *Silences) expire(id string) error {
 		sil.EndsAt = now
 	}
 
-	return s.setSilence(sil, now)
+	return s.setSilence(sil, now, true)
```

Comment:
```suggestion
        // We must skip validation as we expire the silence to avoid not being able to expire a silence that was previously created using Valid UTF-8 characters. 
        // This validation was removed as we allowed the Alertmanager labels names and matchers to be any valid UTF-8 string and can be introduced if necessary once the feature is made permanent.
	return s.setSilence(sil, now, true)
```